extend.php

```diff
@@ -20,61 +20,81 @@
 return [
     (new Extend\Frontend('forum'))
         ->js(__DIR__ . '/js/dist/forum.js')
-        ->css(__DIR__ . '/resources/less/forum.less')
-        ->route('/masquerade/configure', 'masquerade.profile.configure', Content\ConfigureProfile::class)
-        ->route('/masquerade/{username}', 'masquerade.profile.view', Content\ViewProfile::class),
+        ->css(__DIR__ . '/resources/less/forum.less'),
+
     (new Extend\Frontend('admin'))
         ->js(__DIR__ . '/js/dist/admin.js')
         ->css(__DIR__ . '/resources/less/admin.less'),
+
     (new Extend\Routes('api'))
         ->get('/masquerade/fields', 'masquerade.api.fields.index', Api\FieldIndexController::class)
         ->post('/masquerade/fields/order', 'masquerade.api.fields.order', Api\OrderFieldController::class)
         ->post('/masquerade/fields', 'masquerade.api.fields.create', Api\StoreFieldController::class)
         ->patch('/masquerade/fields/{id:[0-9]+}', 'masquerade.api.fields.update', Api\UpdateFieldController::class)
         ->delete('/masquerade/fields[/{id:[0-9]+}]', 'masquerade.api.fields.delete', Api\DeleteFieldController::class)
         ->get('/masquerade/profile/{id:[0-9]+}', 'masquerade.api.profile', Api\UserProfileController::class)
-        ->get('/masquerade/configure', 'masquerade.api.configure', Api\UserConfigureController::class)
-        ->post('/masquerade/configure', 'masquerade.api.configure.save', Api\UserConfigureController::class),
+        ->get('/masquerade/configure/{id:[0-9]+}', 'masquerade.api.configure', Api\UserConfigureController::class)
+        ->post('/masquerade/configure/{id:[0-9]+}', 'masquerade.api.configure.save', Api\UserConfigureController::class),
+
     (new Extend\Middleware('forum'))
         ->add(Middleware\DemandProfileCompletion::class),
+
     (new Extend\Locales(__DIR__ . '/resources/locale')),
 
     (new Extend\ApiController(ShowUserController::class))
         ->addInclude('bioFields.field'),
+
     (new Extend\ApiController(UpdateUserController::class))
         ->addInclude('bioFields.field'),
+
     (new Extend\ApiController(CreateUserController::class))
         ->addInclude('bioFields.field'),
+
     (new Extend\ApiController(ListUsersController::class))
         ->addInclude('bioFields.field'),
+
     (new Extend\ApiController(ListPostsController::class))
         ->addInclude('user.bioFields.field'),
+
     (new Extend\ApiController(ShowDiscussionController::class))
         ->addInclude('posts.user.bioFields.field'),
+
     (new Extend\Model(User::class))
         ->relationship('bioFields', function (User $model) {
             return $model->hasMany(Answer::class)
                 ->whereHas('field', function ($q) {
                     $q->where('on_bio', true);
                 });
         }),
+
     (new Extend\ApiSerializer(BasicUserSerializer::class))
```

Comment:
Not sure if `BasicUserSerializer` is the best place for some of these properties. Perhaps `UserSerializer`?

Reply:
Yes, sorry. This was leftover from when checks were done in the resolver, which would be before the full user record has been fetched from the API.